src/app/@ansyn/ansyn/modules/alerts/model.ts

```diff
@@ -1,6 +1,9 @@
 export enum AlertMsgTypes {
 	OverlaysOutOfBounds = 'overlaysOutOfBounds',
-	overlayIsNotPartOfQuery = 'overlayIsNotPartOfQuery'
+	overlayIsNotPartOfQuery = 'overlayIsNotPartOfQuery',
+	anaglyphSensor = 'anaglyphSensor'
 }
```

Comment:
should we update `alertMsg` Map inside the `overlay-status.reducer` with anaglyphSensor entry?